api/room/schema_query.py

```diff
@@ -369,11 +372,16 @@ def resolve_analytics_ratios(
 
     @Auth.user_roles('Admin', 'Default User')
     def resolve_analytics_ratios_per_room(
-            self, info, start_date, end_date=None):
+            self, info, start_date, end_date=None, room_id=None):
```

Comment:
Is it okay to reduce the number of arguments in such functions. As in use `kwargs` for other arguments and leave only `self` and `info` as positional arguments.